src/tests/admins.test.js

```diff
@@ -1 +1,90 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import Admin from '../models/admins';
+import adminSeed from '../seeds/admins';
+
+const mockAdmin = {
+  firstName: 'admin',
+  lastName: 'admin',
+  dni: 22105409,
+  phone: 1235467855,
+  email: 'ignacio@gmail.com',
+  city: 'Rosario',
+  password: 'Testerfacil123',
+};
+
+const idInvalid = '6465113fb6';
+
+beforeAll(async () => {
+  await Admin.collection.insertMany(adminSeed);
+});
+
+describe('PUT /api/admins', () => {
+  test('should return status 400 dni already exists', async () => {
+    const response = await request(app).put('/api/admins/6465125fb6b5507c22ad8dcd').send({ dni: 33323343 });
+    expect(response.body.message).toBe('There is another admin with that data.');
+    expect(typeof response.body.data).toBe('object');
+    expect(response.status).toBe(400);
+    expect(response.error).toBeTruthy();
+  });
+  test('should return status 400 email already exists', async () => {
+    const response = await request(app).put('/api/admins/6465125fb6b5507c22ad8dcd').send({ firstName: 'admin2', email: 'testing@gmail.com' });
+    expect(response.body.message).toBe('There is another admin with that data.');
+    expect(typeof response.body.data).toBe('object');
```

Comment:
Are we sure that we are testing that our data is empty? I think we could use the following example 
`expect(response.body.data).toEqual({})`

Check if we have to replace something similar throughout the code